xbmc/LangInfo.cpp

```diff
@@ -541,6 +570,45 @@ bool CLangInfo::Load(const std::string& strLanguage)
   }
   g_charsetConverter.reinitCharsetsFromSettings();
 
+  CLog::Log(LOGDEBUG, StringUtils::Format("CLangInfo::Load '{}' '{}'", strLanguage, strFileName));
```

Comment:
from this line and all below imo this is invasive logging, for sure slow devices are impacted
find a way to keep only strictly necessary/minimise
also the use of `StringUtils::Format` is operation not needed since the log already include the format by itself

Reply:
there are examples in our code guidelines https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md

Reply:
Logging level was set to DEBUG and these areas of code are only executed when the Language/Region are loaded, ie, usually once at startup.  I will, however, comment these lines so that they do not compile, but so that future developers can use them for debugging if required.

Reply:
This has been addressed and will be reflected in my next PR update.